en/ui-system/components/editor/canvas.md

```diff
@@ -1,25 +1,22 @@
 # Canvas Component Reference
 
-![canvas](canvas/canvas.png)
+![](canvas/canvas.png)
 
-The node where the **Canvas** component is located is the **root** of the UI rendering. There can be multiple __Canvas__ in the scene. **All UI elements must be placed under a __Canvas__ node so that it can be rendered**. UI Elements are rendered by the **Canvas** node that they are located under. Each **Canvas** can be presented in a certain order controlled by the `priority` attribute.
+The node where the **RenderRoot2D** component is located is the data collection entry point for the 2D rendering component, and the **Canvas** component inherits from the **RenderRoot2D** component, so the **Canvas** component is also the data collection entry point. There can be multiple Canvas nodes in the scene, **all 2D rendering elements must be rendered as children of RenderRoot2D**.
 
-In addition to the root node of the UI, **Canvas** also has a screen adaptation function. Please refer to the [Multi-Resolution Adaption](../engine/multi-resolution.md) documentation to read about this. **Canvas** also supports multi-resolution adaptation. The __design resolution__ and __adaptation scheme__ are uniformly set in the __Project Setting__. A **Camera** is provided inside the __Canvas__, and the default z-axis direction is __-1000 ~ 998__, so the z-axis on the UI must be within this range to display properly (without taking the threshold value).
+In addition to the data entry capability inherited from RenderRoot2D, the Canvas node itself is also an important component for screen adaptation, and plays a key role in multi-resolution adaptation in game production. Please refer to the [Multi-Resolution Adaptation](../engine/multi-resolution.md) documentation. The design resolution and adaptation scheme of Canvas are configured through **Project Settings**.
 
-![design-resolution](canvas/design-resolution.png)
+![](canvas/design-resolution.png)
 
-In the previous design, __Canvas__ was last rendered, meaning it could mask the rendering of all 3D content, but this was far from sufficient for project development needs, such as a 2D map with a 3D character. The **RenderMode** property allows developers to sort the rendering order of the 3D Camera and UI Camera. If you want to have canvas and 3D camera content mixed up, only the lowest camera or canvas can have the **SOLID_COLOR** **ClearFlag**, otherwise a camera with **SOLID_COLOR** flag will erase all content rendered before it.
+The Canvas itself has nothing to do with the camera, its main role is to adapt the screen as described above, so the rendering of the Canvas only depends on the camera corresponding to its node layer, so you can control the properties of the camera to determine the rendering effect of the nodes under the Canvas. 
```

Comment:
```suggestion
The Canvas itself has nothing to do with the camera, its main role is to adapt the screen as described above, so the rendering of the Canvas only depends on the camera corresponding to its node layer, and it can control the properties of the camera to determine the rendering effect of the nodes under the Canvas.
```